### PR TITLE
Update timeout handling in IT.

### PR DESCRIPTION
**Pull request by Yury-Fridlyand**
Note: `@Timeout` annotation uses seconds as a default unit

### CODE REVIEW

File: java/integTest/src/test/java/glide/ErrorHandlingTests.java [hunk 1]
```diff
@@ -12,10 +12,11 @@
 import glide.api.models.exceptions.RequestException;
 import java.net.ServerSocket;
 import java.util.concurrent.ExecutionException;
-import java.util.concurrent.TimeUnit;
 import lombok.SneakyThrows;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Timeout;
 
+@Timeout(20)
```
Comment (jonathanl-bq) [inline java/integTest/src/test/java/glide/ErrorHandlingTests.java:19]: I kind of liked how the old solution told us what units are being used. I thought this was 20 minutes at first glance, until I saw the deleted code.

Comment (aaron-congo) [inline java/integTest/src/test/java/glide/ErrorHandlingTests.java:19]: nit: looks like all the tests previously had a timeout of 10 seconds, should we change this value to 10? Same question for the other files

**Review by aaron-congo — APPROVED** [review]

**Review by SanHalacogluImproving — APPROVED** [review]

File: java/integTest/src/test/java/glide/ErrorHandlingTests.java (hunk 1 above)
Comment (Yury-Fridlyand) [reply java/integTest/src/test/java/glide/ErrorHandlingTests.java]: Fixed

Comment (Yury-Fridlyand) [reply java/integTest/src/test/java/glide/ErrorHandlingTests.java]: I updated the PR description
`@Timeoout` is in seconds

Comment (jonathanl-bq) [reply java/integTest/src/test/java/glide/ErrorHandlingTests.java]: I meant more like if I were to just start reading these tests for the first time, I would not know what the units are. I think a comment might help with clarifying that here.

Comment (Yury-Fridlyand) [reply java/integTest/src/test/java/glide/ErrorHandlingTests.java]: Added

**Review by jonathanl-bq — APPROVED** [review]

**Pull request merged**